packages/vuetify/dev/components.d.ts

```diff
@@ -0,0 +1,78 @@
+    VExpansionPanel: typeof import('@/components/VExpansionPanel/index')['VExpansionPanel']
+    VFooter: typeof import('@/components/VFooter/index')['VFooter']
+    VForm: typeof import('@/components/VForm/index')['VForm']
+    VHover: typeof import('@/components/VHover/index')['VHover']
+    VIcon: typeof import('@/components/VIcon/index')['VIcon']
+    VImg: typeof import('@/components/VImg/index')['VImg']
+    VInput: typeof import('@/components/VInput/index')['VInput']
+    VItem: typeof import('@/components/VItemGroup/index')['VItem']
+    VItemGroup: typeof import('@/components/VItemGroup/index')['VItemGroup']
+    VLabel: typeof import('@/components/VLabel/index')['VLabel']
+    VList: typeof import('@/components/VList/index')['VList']
+    VListGroup: typeof import('@/components/VList/index')['VListGroup']
+    VListItem: typeof import('@/components/VList/index')['VListItem']
+    VMain: typeof import('@/components/VMain/index')['VMain']
+    VMenu: typeof import('@/components/VMenu/index')['VMenu']
+    VMessages: typeof import('@/components/VMessages/index')['VMessages']
+    VNavigationDrawer: typeof import('@/components/VNavigationDrawer/index')['VNavigationDrawer']
+    VOverlay: typeof import('@/components/VOverlay/index')['VOverlay']
+    VPagination: typeof import('@/components/VPagination/index')['VPagination']
+    VProgressCircular: typeof import('@/components/VProgressCircular/index')['VProgressCircular']
+    VProgressLinear: typeof import('@/components/VProgressLinear/index')['VProgressLinear']
+    VRadio: typeof import('@/components/VRadio/index')['VRadio']
+    VRangeSlider: typeof import('@/components/VRangeSlider/index')['VRangeSlider']
+    VRating: typeof import('@/components/VRating/index')['VRating']
+    VResponsive: typeof import('@/components/VResponsive/index')['VResponsive']
+    VSelect: typeof import('@/components/VSelect/index')['VSelect']
+    VSheet: typeof import('@/components/VSheet/index')['VSheet']
+    VSlider: typeof import('@/components/VSlider/index')['VSlider']
+    VSnackbar: typeof import('@/components/VSnackbar/index')['VSnackbar']
+    VSwitch: typeof import('@/components/VSwitch/index')['VSwitch']
+    VSystemBar: typeof import('@/components/VSystemBar/index')['VSystemBar']
+    VTab: typeof import('@/components/VTabs/index')['VTab']
+    VTabs: typeof import('@/components/VTabs/index')['VTabs']
+    VTextarea: typeof import('@/components/VTextarea/index')['VTextarea']
+    VThemeProvider: typeof import('@/components/VThemeProvider/index')['VThemeProvider']
+    VTimeline: typeof import('@/components/VTimeline/index')['VTimeline']
+    VTimelineItem: typeof import('@/components/VTimeline/index')['VTimelineItem']
+    VToolbar: typeof import('@/components/VToolbar/index')['VToolbar']
+    VToolbarTitle: typeof import('@/components/VToolbar/index')['VToolbarTitle']
+    VTooltip: typeof import('@/components/VTooltip/index')['VTooltip']
+  }
+}
+
+export {}
```

Comment:
Just add this to gitignore, otherwise it'll change every time you modify the playground

Reply:
also eslintignore

Reply:
this file is ignored by git so it doesn't bother lint command in CI. Assuming this is for IDE experience, then a `.eslintignore` file must be placed in [root directory](https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file)